### PR TITLE
blog: Add monthly updates post

### PR DESCRIPTION
**Pull request by zapbot**
Signed-off-by: zapbot <12745184+zapbot@users.noreply.github.com>

### CODE REVIEW

**Comment by psiinon** [comment]
Logo
**Checkmarx One – Scan Summary & Details** – [e3009d0f\-e7a7\-43f1\-b276\-8313d4a60c50](https://eu.ast.checkmarx.net/projects/f9121e40-c915-4b13-bb15-36d946943fd8/scans?id=e3009d0f-e7a7-43f1-b276-8313d4a60c50&branch=zapbot%2Fmonthly-updates-blog-post)
### No New Or Fixed Issues Found

**Comment by kingthorin** [comment]
Is there a way to have CX not comment if it has nothing to report?

**Review by kingthorin — COMMENTED** [review]
Nice work @ricekot !!

**Comment by psiinon** [comment]
Ready for review 😁 

<details><summary>Screenshot</summary>
<p>

Screenshot 2024-12-02 at 11-38-12 ZAP – ZAP Updates - November 2024

</p>
</details>

File: site/content/blog/2024-12-02-zap-updates-november-2024/index.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+We would love to hear your [feedback](https://groups.google.com/g/zaproxy-users/c/Sa-9L9lSYd4/m/7wae8ilSAAAJ) on them!
+
+### Using Noir with ZAP
+
+Are you having problems getting ZAP to detect your application endpoints?
+
+If so you should see the [Powering Up DAST with ZAP and Noir](/blog/2024-11-11-powering-up-dast-with-zap-and-noir/)
+blog post, which explains how you can use [OWASP Noir](https://owasp.org/www-project-noir/) to find endpoints in your
+source code and then generate an Open API definition which you can feed into ZAP.
```
Comment (thc202) [inline site/content/blog/2024-12-02-zap-updates-november-2024/index.md:34]: `OpenAPI`

File: site/content/blog/2024-12-02-zap-updates-november-2024/index.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+### Community Links
+
+We updated the [ZAP Team](/docs/team/) pages to include all of the social accounts used for the 
+[ZAP project](/docs/team/zaproxy/) and to include the ZAP Project Manager: [Antero Silva](/docs/team/antero/).
```
Comment (thc202) [inline site/content/blog/2024-12-02-zap-updates-november-2024/index.md:39]: Project -> Product

File: site/content/blog/2024-12-02-zap-updates-november-2024/index.md [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+
+### 2.16.0 Coming Very Soon
+
+The next full ZAP release, 2.16.0 is coming very soon!
+You can follow the [Release 2.16](https://github.com/zaproxy/zaproxy/issues/8706) tracker issue to see exactly how it is progressing.
+
+## New Contributors
+A very warm welcome to the people who started to contribute to ZAP this month!
+
+* [Dragos-Stefan Necula](https://dragosstefannecula.co.uk">dragosstefannecula.co.uk)
```
Comment (thc202) [inline site/content/blog/2024-12-02-zap-updates-november-2024/index.md:51]: The link needs update.

**Comment by psiinon** [comment]
Fixed

**Comment by thc202** [comment]
Thank you!

File: site/content/blog/2024-12-02-zap-updates-november-2024/index.md [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+authors:
+- psiinon
+---
+
+## Highlights
+
+### Scan Policies Add-On
+
+This month we released a new [Scan Policies](/docs/desktop/addons/scan-policies/) add-on which provides a set of
+Scan Policies tuned for different purposes.
```
Comment (kingthorin) [inline site/content/blog/2024-12-02-zap-updates-november-2024/index.md:21]: ...purposes and audiences.

or "personas" ??

**Comment by psiinon** [comment]
@kingthorin  / @ricekot ?

File: site/content/blog/2024-12-02-zap-updates-november-2024/index.md (hunk 4 above)
Comment (psiinon) [reply site/content/blog/2024-12-02-zap-updates-november-2024/index.md:21]: Copied directly from https://www.zaproxy.org/docs/desktop/addons/scan-policies/ 😁

Comment (kingthorin) [reply site/content/blog/2024-12-02-zap-updates-november-2024/index.md:21]: Okay, not worth blocking on.

**Review by kingthorin — APPROVED** [review]

**Pull request merged**